docs/querying/sql.md

```diff
@@ -956,6 +955,24 @@ trailer they all include: one blank line at the end of the result set. If you de
 through a JSON parsing error or through a missing trailing newline, you should assume the response was not fully
 delivered due to an error.
 
+### HTTP DELETE
+You can use the HTTP `DELETE` method to cancel a SQL query using the following syntax:
+`DELETE https://BROKER:8082/druid/v2/sql/{sqlQueryId}`. The DELETE method requires the `sqlQueryId`, therefore to predict the query id you must set it in the query context.
```

Comment:
@jihoonson do we want to say something about what happens behind the scenes with this query cancellation?